sdk/data/azcosmos/cosmos_container.go

```diff
@@ -75,6 +79,9 @@ func (c *ContainerClient) Replace(
 	ctx context.Context,
 	containerProperties ContainerProperties,
 	o *ReplaceContainerOptions) (ContainerResponse, error) {
+	var err error
+	ctx, endSpan := runtime.StartSpan(ctx, "ContainerClient.Replace", c.database.client.internal.Tracer(), nil)
```

Comment:
@jhendrixMSFT do you happen to remember if we support named tracers in Go?
In other words, what would be the name of this tracer? 
Also, what would be the way to specify schema_url (it contains semantic convention version) for it? 

(excuse my Go ignorance 😳)

Reply:
> For named tracers, are you talking about this or something else?

I mean here

https://github.com/Azure/azure-sdk-for-go/blob/969cfcd7a36018cab9461bc768bdd0c8f1ff4e00/sdk/tracing/azotel/otel.go#L31

when we create a tracer, what would be the value of the namespace? 

> schema_url is set by [azotel](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/tracing/azotel/otel.go#L31) when the adapter is created. Do we need SDKs to set it?

Yes, I think so - cosmos and servicebus may generate different schema versions and should be able to express it - I'll create an issue to follow up - it's not blocking this PR.

Reply:
created https://github.com/Azure/azure-sdk-for-go/issues/23307